api-gateway/src/main/resources/static/scripts/visits/visits.template.html

```diff
@@ -705,6 +723,10 @@ <h3 style="margin-top: 6px">Added Visits</h3>
         <td style="text-align:center; vertical-align:middle"><button class="btn btn-danger" ng-click="$ctrl.showConfirmationModal($event, v.visitId)">Delete visit</button></td>
     </tr>
     </tbody>
+<<<<<<< HEAD
```

Comment:
[nitpick] For this, I'm guessing that you just forgot to remove it or perhaps didn't see it. It's always good to keep the code as clean as possible in order to make it readable.